lib/npm-install.js

```diff
@@ -30,7 +30,7 @@ function npmInstall(settings) {
   return new Promise((resolve, reject) => {
     let error = '';
 
-    // This doesn't exist if we're logging verbosely
+    // stderr property doesn't exist is spawned with stdio === 'inherit'
```

Comment:
Should the "is" be "if"?

Reply:
Let me have my typos and be gone.  JK, fixed it.  :-)